kyuubi-server/src/main/scala/org/apache/kyuubi/engine/KyuubiApplicationManager.scala

```diff
@@ -115,7 +115,9 @@ object KyuubiApplicationManager {
   }
 
   private def setupFlinkYarnTag(tag: String, conf: KyuubiConf): Unit = {
-    val originalTag = conf.getOption(FlinkProcessBuilder.YARN_TAG_KEY).map(_ + ",").getOrElse("")
+    val originalTag = conf.getOption(s"flink.${FlinkProcessBuilder.YARN_TAG_KEY}")
```

Comment:
Could we avoid the hardcode `flink.` prefix?

Reply:
Is this change compatible with Kyuubi's kill-application-by-tag mechanism?

Reply:
> Could we avoid the hardcoded flink. prefix?

Sure, I will add a constant for it.

> BTW is this change compatible with Kyuubi's kill-application-by-tag mechanism?

It is compatible, the previous logic also merges user-specified tags, I just added the key prefixed by `flink.`.